app/models/global_state.rb

```diff
@@ -58,4 +61,16 @@ def self.valid_new_incident_years
     # for last year iff submission hasn't closed out for the year.
     submission_closed_for_year? ? [Time.current.year] : [Time.current.year - 1, Time.current.year]
   end
+
+  def self.maintenance_mode?
+    instance.maintenance_mode
```

Comment:
1. This PR requires a migration (plz write and include here).
2. This method should gracefully handle the pre-migration case, where 'maintenance_mode' may not be an attribute. Just have it return false in those cases, then when we run the migration it can start picking up the new values.